src/main/java/io/stargate/sgv2/jsonapi/exception/playing/ErrorScope.java

```diff
@@ -0,0 +1,25 @@
+package io.stargate.sgv2.jsonapi.exception.playing;
+
+/**
+ * Interface for any enum that represents an error scope to implement.
+ *
+ * <p>This is used to group errors together in a {@link ErrorFamily}.
+ *
+ * <p>This interface is used because multiple ENUM's define the scopes, the interface creates a way
+ * to treat the values from the different ENUM's in a consistent way.
+ *
+ * <p>See {@link APIException}
+ */
+@FunctionalInterface
+public interface ErrorScope {
```

Comment:
I don't understand this -- why not just a simple Enum? What is the point in this contraption?

Reply:
discussed - make it clear this is here because the scope is  disjoint sets for the diff families